server/src/main/java/org/apache/druid/metadata/PendingSegmentRecord.java

```diff
@@ -33,11 +33,14 @@
 /**
  * Representation of a record in the pending segments table. <br/>
  * Mapping of column in table to field:
- * <li/>  id -> id (Unique identifier for pending segment)
- * <li/>  sequence_name -> sequenceName (sequence name used for segment allocation)
- * <li/>  sequence_prev_id -> sequencePrevId (previous segment id used for segment allocation)
- * <li/>  upgraded_from_segment_id -> upgradedFromSegmentId (Id of the root segment from which this was upgraded)
- * <li/>  task_allocator_id -> taskAllocatorId (Associates a task / task group / replica group with the pending segment)
+ *
+ * <ul>
+ * <li>  id -> id (Unique identifier for pending segment) <li/>
+ * <li>  sequence_name -> sequenceName (sequence name used for segment allocation) <li/>
+ * <li>  sequence_prev_id -> sequencePrevId (previous segment id used for segment allocation) <li/>
+ * <li>  upgraded_from_segment_id -> upgradedFromSegmentId (Id of the root segment from which this was upgraded) <li/>
+ * <li>  task_allocator_id -> taskAllocatorId (Associates a task / task group / replica group with the pending segment) <li/>
```

Comment:
This list is not needed here, the description of the fields should be in the javadocs of the respective getters.